project/Game.py

```diff
@@ -70,10 +62,11 @@ def __init__(self):
 
         def add_team(self, name, password):
             if not self.started:
-                team = TeamFactory().getTeam(name, password)
-                if team in self.teams:
-                    return False
-                self.teams.append(team)
+                teamToBeAdded = TeamFactory().getTeam(name, password)
+                for team in self.teams:
+                    if team.get_username() == teamToBeAdded.username:
```

Comment:
I think we should look into overriding \_\_eq\_\_ so that we can just check if teamToBeAdded in self.teams. Since teams don't need to be ordered we could also look into moving those to a set so there can't be duplicates (need the \_\_eq\_\_ change before we can look at set)